deprecated/classes/subs-cpt.php

```diff
@@ -379,7 +379,7 @@ public function sort_columns( $vars ) {
 	 */
 	public function custom_columns( $column, $sub_id ) {
 		if ( isset ( $_GET['form_id'] ) ) {
-			$form_id = $_GET['form_id'];
+			$form_id = absint($_GET['form_id']);
 			if ( $column == 'id' ) {
 				echo apply_filters( 'nf_sub_table_seq_num', Ninja_Forms()->sub( $sub_id )->get_seq_num(), $sub_id, $column );
```

Comment:
Thanks Brett, does this matter considering this is in the "deprecated" area of the plugin? I'm curious about that. Happy to change though.

Reply:
I don't believe this matters too much. The "deprecated" bit of the plugin is loaded based on the `ninja_forms_load_deprecated` option [here](https://github.com/NBCUniversal-OTS/ninja-forms/blob/1dd4737b07015bce1aa5fba59e3d0fe6a5465657/ninja-forms.php#L33) so I don't think it will ever be loaded in your instance.